### DISCUSSION TITLE
Live Prompt Realtime Diffusion Demo Video

### DISCUSSION THREAD

**Discussion by TheyCallMeHex** [saddam213/OnnxStack#40] [Show and tell]
A preview demo video of the Live Prompt realtime diffusion feature being added to OnnxStack in an upcoming release:

https://www.youtube.com/watch?v=U1qL-XuHT5o

**Comment by saddam213** [comment]
Live Paint is amazing

![image](https://github.com/saddam213/OnnxStack/assets/4353684/72940712-07c4-4c53-b158-076f71dd860a)
